framework/src/task_manager/task_manager_core.c

```diff
@@ -1495,6 +1500,20 @@ static int taskmgr_init_task_manager(void)
 	return OK;
 }
 
+void task_manager_populate_builtin_list(const builtin_info_t builtin_list[], int builtin_list_count)
+{
+	tm_builtin_list = TM_ALLOC(builtin_list_count * sizeof(tm_builtin_info_t));
+	for (int i = 0; i < builtin_list_count; i++) {
+		strncpy(tm_builtin_list[i].name, builtin_list[i].name, CONFIG_TASK_NAME_SIZE);
+		tm_builtin_list[i].entry = builtin_list[i].entry;
+		tm_builtin_list[i].exectype = builtin_list[i].exectype;
+		tm_builtin_list[i].priority = builtin_list[i].priority;
+		tm_builtin_list[i].stacksize = builtin_list[i].stacksize;
+	}
+
+	builtin_list_cnt = builtin_list_count;
```

Comment:
Possible to make these two variable names a bit more distinct?
"cnt" vs "count" could be confusing in the future.

Reply:
Changed the global variable name to tm_builtin_list_cnt

Reply:
Changed the global variable name to tm_builtin_list_cnt .